client/cmdhf15.c

```diff
@@ -945,7 +956,8 @@ static int CmdHF15FindAfi(const char *Cmd) {
 
     clearCommandBuffer();
     SendCommandMIX(CMD_HF_ISO15693_FINDAFI, strtol(Cmd, NULL, 0), 0, 0, NULL, 0);
-    return 0;
+    DropField();
```

Comment:
This one is harder,   the FINDAFI command doesn't report back when it finished running.
So doing this, would interrupt the device side procedure aka command will not bruteforce AFi..

Reply:
Yes, I thought so too, but I tried it after that change and it did find a couple of AFIs. But maybe it was just pure luck it found those.

Reply:
I think the solution is for the command to send a message back to the client once its done.
and have the client loop / wait for that response.

Reply:
better to loop 2sec waittimeout,  look in "hf mf darkside" code for instance.   It will give a unified command execution.